scripts/o3de/o3de/engine_template.py

```diff
@@ -2300,6 +2299,75 @@ def create_gem(gem_path: pathlib.Path,
     # Register the gem with the either o3de_manifest.json, engine.json or project.json based on the gem path
     return register.register(gem_path=gem_path) if not no_register else 0
 
+def create_repo(repo_path: pathlib.Path,
+                repo_uri: str,
+                repo_name: str = None,
+                origin: str = None,
+                origin_url: str = None,
+                summary: str = None,
+                additional_info: str = None,
+                force: bool = False,
+                replace: list = None
+                ) -> int:
+
+    template_name = 'RemoteRepo'
+
+    template_path = manifest.get_registered(template_name=template_name)
+
+    template_json = template_path / 'template.json'
+  
+    # read in the template.json
+    with open(template_json) as s:
+        try:
+            template_json_data = json.load(s)
+        except KeyError as e:
+            logger.error(f'Could not read template json {template_json}: {str(e)}.')
+            return 1
+
+    if not repo_path:
+        logger.error('Destination path on local machine cannot be empty.')
+        return 1
+    if not repo_uri:
+        logger.error('Destination path on online version control uri cannot be empty.')
+        return 1
+    if not repo_path and repo_uri:
+        logger.error('Please provide repository destination path on local machine and online uri.')
+        return 1
+    
+    if not force and os.path.isdir(repo_path):
+        logger.error(f'Destination path {repo_path} already exists.')
+        return 1
+    else:
+        os.makedirs(repo_path, exist_ok=force)
+
+    if not repo_name:
+        # destination name is now the last component of the destination_path
+        repo_name = repo_path.name
+
+    # any user supplied replacements
+    replacements = list()
+    while replace:
+        replace_this = replace.pop(0)
+        with_this = replace.pop(0)
+        replacements.append((replace_this, with_this))
+
+    replacements.append(("${Name}", repo_name))
+    replacements.append(("${RepoURI}", repo_uri))
+    replacements.append(("${Origin}",  origin if origin else "${Origin}"))
+    replacements.append(("${OriginURL}",  origin_url if origin_url else "${OriginURL}"))
+    replacements.append(("${Summary}", summary if summary else "${Summary}"))
+    replacements.append(("${AdditionalInfo}", additional_info if additional_info else "${AdditionalInfo}"))
+
+    # create repo.json file
+    if _execute_template_json(template_json_data,
+                              repo_path,
+                              template_path,
+                              replacements):    
+        logger.error(f'Instantiation of the remote repository has failed.')
+        return 1
```

Comment:
`_execute_template_json` does not appear to return any value

```suggestion
    _execute_template_json(template_json_data, repo_path, template_path, replacements)
```